packages/perseus/src/renderer.tsx

```diff
@@ -853,15 +856,11 @@ class Renderer extends React.Component<Props, State> {
         ];
     };
 
-    getWidgetInstance: (id: string) => Widget | null | undefined = (
-        id: string,
-    ): Widget | null | undefined => {
```

Comment:
With arrow functions on classes, we only need the Typescript typing on the first part (no idea what it's called)... the arrow function definition after the `=` correctly infers the types if we properly type the member.

Reply:
Yeah, you could do this either way, but since the function is so strongly defined in itself deleting the definition before "=" would work great as well!